sdk/lib/core/regexp.dart

```diff
@@ -30,6 +30,13 @@ part of dart.core;
  * Iterable<Match> matches = exp.allMatches(str);
  * ```
  *
+ * The following example finds the content of the first matching group.
+ * ```dart
+ *  RegExp exp = new RegExp("/transactions/([0-9]+)/invoices");
```

Comment:
Alternative way to write the RegExp: `r"/transactions/(\d+)/invoices"`
(Either is fine, I just always start wondering "why" when I see `[0-9]` instead of `\d`).